src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -474,9 +474,13 @@ g.test('texture_size,default_value_and_smallest_size,compressed_format')
           { size: [blockWidth, blockHeight, 1], _success: true },
         ];
       })
+      // Filter if the dimension is 3D and format is not compatible
+      .filter(
+        ({ dimension, format }) => !(dimension === '3d' && !kTextureFormatInfo[format].texture3D)
+      )
```

Comment:
nit (IMO easier to read):

```suggestion
      .unless(
        ({ dimension, format }) => dimension === '3d' && !kTextureFormatInfo[format].texture3D
      )
```